File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.java
class Solution {
    public int solution(int a, int b) {
        int answer;
        String s = String.valueOf(a) + String.valueOf(b);
        int i = Integer.valueOf(s);
        if(i >= (2*a*b)){
            answer = i;
        }else{
            answer = (2*a*b);
        }
        return answer;
    }
}

File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.py
def solution(a, b):
    
    # 1)
    # A = int(str(a) + str(b))
    # B = 2 * a * b
    # answer = A if A >= B else B
    
    # 2)
    answer = max(int(str(a) + str(b)), 2 * a * b)
    
    return answer